### PR TITLE
Updating and Linting MoPub Adapter to version 5.4.0.0.

### PR DESCRIPTION
**Pull request by maninomula**

### CODE REVIEW

**Comment by mopubmediation** [comment]
@maninomula I reviewed and find that this PR does not have any code changes. All the changes are related to formatting. Can you please confirm?

File: adapters/MoPub/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
 # MoPub Ads Mediation Adapter for Google Mobile Ads SDK for iOS
 
+## Version 5.4.0.0
+- Verifed compatibility with MOPub SDK 5.4.0.
```
Comment (mopubmediation) [inline adapters/MoPub/CHANGELOG.md]: `MOPub` --> `MoPub`

**Comment by maninomula** [comment]
Yes, All the changes are related to formatting but just want to use the workflow we are going to follow for the next updates. Thanks for pointing out the correction in 'CHANGELOG.md'.

**Comment by maninomula** [comment]
Thanks @mopubmediation for the review.

**Pull request merged**